src/pages/solve/Knapsack.tsx

```diff
@@ -0,0 +1,39 @@
+import { Heading, Text } from "@chakra-ui/react";
+import { NextPage } from "next";
+import { useState } from "react";
+import { Layout } from "../../components/layout/Layout";
+import { SolverConfiguration } from "../../components/solvers/SolverConfiguration";
+import { TextInputMask } from "../../components/solvers/TextInputMask";
+
+const Knapsack: NextPage = () => {
+  const [knapsackProblem, setKnapsackProblem] = useState("");
+
+  return (
+    <Layout>
+      <Heading as="h1">Knapsack Solver</Heading>
+
+      <Text color="text" align="justify">
+        The Knapsack problem is a combinatorial optimization problem that seeks
+        to maximize the total value of items in a knapsack without exceeding the
+        capacity of the knapsack. You can enter a problem in the following
```

Comment:
The format description is currently hard to understand. Also, we want to provide example problems.
Please create a new Repository (in the ProvideQ orga) names ProvideQ/knapsack-examples. Add some example problems there. (Different sized problems (maybe 5, 20, 50, 100, and 200 items), gives them meaningful names like: 20-items-knapsack.txt)

You can find code to generate knapsack problems here: http://hjemmesider.diku.dk/~pisinger/codes.html

Inside of this Repository you should add a Readme that describes the knapsack format in Detail (what is currently done here). You could also add one or two small examples to the Readme

Reply:
added repo [knapsack-problems](https://github.com/ProvideQ/knapsack-problems) (-problems instead of -examples for uniformity with already existing repo [maxcut-examples](https://github.com/ProvideQ/maxcut-problems).